drivers/flash/Kconfig.stm32_ospi

```diff
@@ -24,3 +24,27 @@ config FLASH_STM32_OSPI
 					$(DT_STM32_OCTOSPI_HAS_DMA)
 	help
 	  Enable OSPI-NOR support on the STM32 family of processors.
+
+if FLASH_STM32_OSPI
+
+if SOC_STM32U595XX || SOC_STM32U595XX || SOC_STM32U595XX || SOC_STM32U595XX \
+	|| SOC_STM32U595XX || SOC_STM32U595XX || SOC_STM32U595XX || SOC_STM32U595XX \
+	|| SOC_STM32U595XX || SOC_STM32U595XX || SOC_STM32U599XX || SOC_STM32U599XX \
+	|| SOC_STM32U599XX || SOC_STM32U599XX || SOC_STM32U599XX || SOC_STM32U599XX \
+	|| SOC_STM32U599XX || SOC_STM32U5A5XX || SOC_STM32U5A5XX || SOC_STM32U5A5XX \
+	|| SOC_STM32U5A5XX || SOC_STM32U5A5XX || SOC_STM32U5A9XX || SOC_STM32U5A9XX \
+	|| SOC_STM32U5A9XX || SOC_STM32U5A9XX
```

Comment:
can you review this list ?

Reply:
grepping trough the zephyr source gives me following list:
```
SOC_STM32U595XX
SOC_STM32U599XX
SOC_STM32U5A5XX
SOC_STM32U5A9XX
```
should I limit it to SOCs mentioned in the source tree?